CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+# 1.7.0-alpha02 (July 2024)
+_Changes since 1.7.0-alpha01_
+
+## Breaking Changes
+
+### Multiple Platforms
+- [ComposeScene `fun hitTestInteropView(position: Offset): Boolean` changed to `fun hitTestInteropView(position: Offset): InteropView?`](https://github.com/JetBrains/compose-multiplatform-core/pull/1426)
```

Comment:
`ComposeScene` is internal API and shouldn't be mentioned here. Especially as "breaking change"